components/stepper/overview.md

```diff
@@ -53,18 +48,56 @@ To use a Telerik Stepper for Blazor:
 }
 ````
 
-## Features
+## Steps
+
+The Steps are the building blocks of the Stepper component. You can include the desired steps by adding a `StepperStep` tag for every step. The steps appearance can be configured through the features the `StepperStep` exposes. [Read more about the Blazor Stepper configuration]({%slug stepper-steps-overview%}).
+
+## Orientation
+The Stepper component provides different orientations. [Read more about the Blazor Stepper orientation]({%slug stepper-orientation%}).
+
+## Display Modes
+
+The Stepper component provides different display modes. [Read more about the Blazor Stepper display modes]({%slug stepper-display-modes%}).
+
+## Linear Flow
+
+By default, the user can select any step in the Stepper component. This behavior could be customized by configuring its Linear flow property. [Read more about the Blazor Stepper linear flow]({%slug stepper-linear-flow%}).
 
-The Stepper provides the following features:
+## Templates
 
-* `Orientation` - `StepperOrientation` - defines the orientation of the Stepper. You can read more on how to configure it in the [Orientation]({%slug stepper-orientation%}) article.
+You can use the functionality of the built-in templates and customize what is rendered as steps. [Read more about the Blazor Stepper templates]({%slug step-template%}).
 
-* `Value ` - `int`- defines the current step index.
+## Events
 
-* `Linear` - `bool` - enables/disables [linear flow]({%slug stepper-linear-flow%}).
+The Blazor Stepper generates events that you can handle and further customize its behavior. [Read more about the Blazor Stepper events]({%slug stepper-events%}).
 
-* `StepType` - `StepperStepType` - defines the display mode of the Stepper. You can read more on how to configure it in the [Display modes]({%slug stepper-display-modes%}) article.
+## Parameters
+
+The Blazor Stepper provides various parameters that allow you to configure the component:
+
+<style>
+    article style + table {
+        table-layout: auto;
+        word-break: normal;
+    }
+</style>
+| Parameter | Type | Description |
+| ----------- | ----------- | ----------- |
+| `Value` | `int` | Defines the current step index. |
+| `Orientation` | `StepperOrientation` enum <br /> `Horizontal` | Defines the orientation of the Stepper. You can read more on how to configure it in the [Orientation]({%slug stepper-orientation%}) article. |
+| `Linear` | `bool` | Enables/disables [linear flow]({%slug stepper-linear-flow%}). |
+| `StepType` | `StepperStepType` enum <br /> `Steps` | Defines the display mode of the Stepper. You can read more on how to configure it in the [Display modes]({%slug stepper-display-modes%}) article. |
+
+## Next Steps
+
+* [Using Stepper Templates]({%slug step-template%})
+
+* [Configure Steps]({%slug stepper-steps-overview%})
+
+* [Explore the Stepper Events]({%slug stepper-events%})
```

Comment:
The Next Steps order should make sense. First you configure steps, then you use templates. Use the same grammar form for the verbs.

```suggestion
* [Configure Steps]({%slug stepper-steps-overview%})
* [Use Stepper Templates]({%slug step-template%})
* [Explore Stepper Events]({%slug stepper-events%})
```